admin/src/js/controllers/edit-user.js

```diff
@@ -298,7 +298,14 @@ angular.module('controllers').controller('EditUserCtrl',
                   $uibModalInstance.close();
                 })
                 .catch(function(err) {
-                  $scope.setError(err, 'Error updating user');
+                  if (err.status === 400) {
```

Comment:
Now that we have the `translation_key` field, maybe we should check if it exists instead of the status.

Reply:
I think the status should still be checked just incase the error returned isn't a 400 error.